streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -816,26 +815,4 @@ public void shouldRecordTaskClosedMetricOnClose() throws IOException {
         final double expectedCloseTaskMetric = 1.0;
         verifyCloseTaskMetric(expectedCloseTaskMetric, streamsMetrics, metricName);
     }
-
-    @Test
-    public void shouldRecordTaskClosedMetricOnCloseSuspended() throws IOException {
```

Comment:
We would like to restore this test if we choose to keep standby tasks metrics.

Reply:
Removed the test because we no longer suspend standbys (we still have a test for recording metrics on task close)